src/lib/components/AlignmentViewer.react.js

```diff
@@ -1,295 +0,0 @@
-import React, {Component} from 'react';
-import PropTypes from 'prop-types';
-
-import {omit} from 'ramda';
-
-import {AlignmentViewer as PreAlignementViewer} from 'react-alignment-viewer';
-
-/**
- * The Alignment Viewer (MSA) component is used to align multiple genomic
- * or proteomic sequences from a FASTA or Clustal file. Among its
- * extensive set of features, the multiple sequence alignment viewer
- * can display multiple subplots showing gap and conservation info,
- * alongside industry standard colorscale support and consensus sequence.
- * No matter what size your alignment is, Aligment Viewer is able to display
- * your genes or proteins snappily thanks to the underlying WebGL architecture
- * powering the component. You can quickly scroll through your long sequence
- * with a slider or a heatmap overview.
- * Note that the AlignmentChart only returns a chart of the sequence, while
```

Comment:
This docs looks a bit confusing to me.Same as in Alignment Chart which we'll keep https://github.com/plotly/dash-bio/blob/447ebbe01d716a0d652e744d8fe44b781fa66ad4/src/lib/components/AlignmentChart.react.js#L17-L19

Does this `Alignment Viewer` here refers to original react component? I didn't go detailed into the difference between this two components but we could skip mentioning the one we won't be using as it'll be packed into docstring reference.

Reply:
Good catch! Will fix this ASAP :)